packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -7,6 +7,7 @@ import { ApiClient } from '../.'
  * - 'all' to list the whole DB
  */
 type OwnerType = 'mine' | 'all' | string
+type TokenType = `${string}@${string}`
```

Comment:
Added this

Reply:
this is actually called `account` in blockchain